test/lib/ufe/testAttribute.py

```diff
@@ -70,6 +75,19 @@ def setUpClass(cls):
 
         random.seed()
 
+        # Test if Maya's getAttr command supports ufe.
+        if cls.pluginsLoaded:
+            import mayaUsd_createStageWithNewLayer
+            psPathStr = mayaUsd_createStageWithNewLayer.createStageWithNewLayer()
+            stage = mayaUsd.lib.GetPrim(psPathStr).GetStage()
+            stage.DefinePrim('/A', 'Xform')
+            try:
+                cmds.getAttr('|stage1|stageShape1,/A.visibility')
+                cls.getAttrSupportsUfe = True
+            except:
+                pass
```

Comment:
I guess to get rid of the bandit security warning you could write
cls.getAttrSupportsUfe = False
here, even though it's redundant.